nasa/harmony
# DAS-2071 - Migrate to GitHub-hosted HyBIG image.

## Jira Issue ID

DAS-2071

## Description

This PR updates the Docker image that Harmony will use for the Harmony Browse Image Generator (HyBIG) to use [the new one hosted in the GitHub Container Registry](https://github.com/nasa/harmony-browse-image-generator/pkgs/container/harmony-browse-image-generator).

## Local Test Steps

* Pull this branch of Harmony locally.
* Pull the Docker image for HyBIG from GitHub, or build it locally from the HEAD of `main` for the HyBIG repository.
* Run through [the HyBIG regression tests](https://github.com/nasa/harmony-regression-tests/tree/main/test/hybig) locally, to make sure the service gets invoked.

## PR Acceptance Checklist
* ~~Tests added/updated (if needed) and passing~~
* [x] Documentation updated (if needed) - updated the changelog for environment variables.
